### DISCUSSION TITLE
How to add UPS in Netbox

### DISCUSSION THREAD

**Discussion by x3m** [netbox-community/netbox#6890] [Q&A] [answered]
Hi all,

I'm new to Netbox... was wondering what is the right way to add UPS/Inverter as device in the rack, with N outlets, and then connect Switch PSU1 (for example) to Outlet X on the UPS? I didn't see native way to add UPS, so what is best practice advise?

For Inverter specifically I will most probably need to add type/size of batteries as they usually have external batteries that can be different size/model.

Thanks in advance

**Comment by candlerb** [comment, marked as answer]
A UPS is just a device with (usually one) power port, and (N) power outlets.  You associate each of the power outlets to the power port, when adding the power outlets to the device type or device.

If you mentally replace "power port" with "power inlet", it may be clearer.

To connect to the switch, just add a cable between one of the UPS power outlets, and the switch power port.

The batteries can be added as Inventory Items.

**Comment by x3m** [comment]
Thanks, let me try
